addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -469,7 +469,7 @@ def _format_transfer_source_log(self, balances_per_account, transfer_move):
 
         return Markup("<ul>%s</ul>") % Markup().join([
             Markup("<li>%s</li>") % \
-                self._format_strings(transfer_format, transfer_move, balance) % (Markup("<strong>%s</strong>") % account.display_name)
+                self._format_strings(transfer_format % (Markup("<strong>%s</strong>") % account.display_name), transfer_move, balance)
```

Comment:
It could be good if you did like here https://github.com/odoo/odoo/pull/144146/files#diff-481f9e580acb354ddf0eece22a3587edb8ad5b8616b283c667a5054f2729b7d1R449
It can be more readable.

Reply:
Yes, I thought of that BUT as mentioned in the discussion above, that would break the existing translations. That's why I stuck with this ugly but rather stable implementation

Reply:
here's the translation I am referring to https://github.com/odoo/odoo/blob/f744e90fc4fa05c224194a80cb44b01734fac081/addons/account/i18n/account.pot#L16596-L16603

Reply:
Yes, I'm aware of what translations are. I was looking at .po's, and indeed there are existing ones, I misread.
You still need to retarget to 15.0

Reply:
@gawa-odoo I rebased it and adjusted the fix for **15.0**